.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -25,32 +25,23 @@ assignees: ''
  If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
  Please tag fewer than 3 people.
 
- albert, bert, GPT2, XLM: @LysandreJik
- tokenizers: @mfuntowicz
- Trainer: @sgugger
- Speed and Memory Benchmarks: @patrickvonplaten
- Model Cards: @julien-c
- TextGeneration: @TevenLeScao
+ albert, bert, gpt2, xlm: @LysandreJik
+ bart, blenderbot, t5, speed and memory benchmarks, pegasus, longformer/reformer, marian, mbart: @patrickvonplaten
```

Comment:
```suggestion
 bart, blenderbot, t5, pegasus, marian, mbart: @patrickvonplaten, @patil-suraj 
 longformer, reformer, speed and memory benchmarks: @patrickvonplaten
```

Reply:
oh, we are working on the PR template and then will sync the two, so let's focus on the former in this review. thanks.

I assume the two should be identical. Please correct me if I'm wrong.

Reply:
Yes, the two should be identical IMO